ember/app/routes/projects/index.js

```diff
@@ -5,9 +5,10 @@ export default class ProjectsRoute extends Route {
   @service store;
 
   model() {
+    this.store.unloadAll('project');
```

Comment:
Put this in the before model hook.

Reply:
done here: https://github.com/adfinis/Outdated/commit/c965aee52e37ef22d4143f91c193b9f67274973a#diff-05025f83ba1ff583208b0467b8038cf8402004c1e65b2983413bbd375f5d921cL4-R16